fjord/base/tests/test_auth.py

```diff
@@ -20,17 +22,18 @@ def test_new_user(self):
         # FIXME - this can go away post django-browserid 0.9
         new_user.backend = 'django_browserid.auth.BrowserIDBackend'
```

Comment:
Given that we're fiddling in here, can you see if this FIXME is right and that we can nix that line?